stacks-common/src/types/mod.rs

```diff
@@ -152,6 +152,12 @@ impl StacksEpochId {
         }
     }
 
+    /// Is a timestamp saved for each Stacks block in this epoch?
+    /// If not, the timestamp is retrieved from the burn block.
+    pub fn has_block_timestamps(&self) -> bool {
```

Comment:
This function name is a bit confusing.  Can you call it something more specific, like `has_stacks_block_timestamps()` or `has_independent_block_timestamps()`?  All Stacks blocks have timestamps; in Nakamoto, they happen to be independent of the Bitcoin block.

Reply:
Changed in 67f0ae2ab34b398bba4d879926fd24e771b1f487

Reply:
Actually, I changed this in 0811895a9bbe19ceeb7d95eef4586a22172bf6dc to be more generic `uses_nakamoto_blocks`, then I can also use it to decide which headers table to use.